composer.json

```diff
@@ -21,17 +21,17 @@
     "prefer-stable": true,
     "require": {
         "php": "^7.4|^8.0",
-        "yiisoft/db": "^3.0@dev",
```

Comment:
Shouldn't we still specify `@dev?`

Reply:
It's an aliases it works both ways without problems, but I fix it now.

Reply:
done.